Docs/Configuration.tex

```diff
@@ -1850,13 +1849,13 @@ \subsection{Properties}\label{kernelprops}
   \textbf{Description}: Remove selected kernel drivers from prelinked kernel.
 
   Designed to be filled with \texttt{plist\ dictionary} values, describing each
-  blocked driver. See \hyperref[kernelpropsblock]{Block Properties} section below.
+  blocked driver. See the \hyperref[kernelpropsblock]{Block Properties} section below.
 
 \item
   \texttt{Emulate}\\
   \textbf{Type}: \texttt{plist\ dict}\\
-  \textbf{Description}: Emulate select hardware in kernelspace via parameters
-  described in \hyperref[kernelpropsemu]{Emulate Properties} section below.
+  \textbf{Description}: Emulate certain hardware in kernelspace via parameters
```

Comment:
I am not sure certain → select is a good change oretty much everywhere. We do not know the exact set behind the description, so select is more accurate.

Reply:
Hi Vit,

Not sure if I should just butt out of the review process now?

But as another born, native speaker, I do agree with @dakanji here - 'select' in English I think perhaps does not carry the exact connotations which you think it carries?

'certain items': a subset of the total set of items, chosen according to, well, certain criteria (i.e. criteria which exist and are definite, even though not explicitly specified here)

'select items': choice or premium items, chosen according to the criterion of being better, in some way, than the rest of the items

Reply:
"Certain" does not say you know the exact set and is the appropriate term here. A fancy term for "Some" in the context.

You typically use "Select" when you want to imply exclusivity. A slightly similar term is "Curated"